openedx/core/djangoapps/programs/utils.py

```diff
@@ -3,13 +3,16 @@
 import datetime
 import logging
 from urlparse import urljoin
+from pytz import utc
+
+from xmodule.fields import Date
 
 from django.conf import settings
 from django.core.urlresolvers import reverse
 from django.utils import timezone
 from django.utils.text import slugify
 from opaque_keys.edx.keys import CourseKey
-import pytz
+from django.utils.translation import ugettext
```

Comment:
Again, I don't expect this to stay here, but if it does it should be grouped alphabetically with the other django imports in this module.